src/compiler/diagnosticMessages.json

```diff
@@ -5056,6 +5056,10 @@
         "category": "Message",
         "code": 90034
     },
+    "Initialize a private property '{0}'": {

```

Comment:
It's not really the initialization, it's more the declaration

```
Declare private field '{0}'.
```

Reply:
@DanielRosenwasser Changed.